addons/aircraft/CfgAmmo.hpp

```diff
@@ -51,8 +51,25 @@ class CfgAmmo {
 
     // adjust damage and splash damage, closer to bluefor gatling with same caliber
     class Cannon_30mm_HE_Plane_CAS_02_F: Gatling_30mm_HE_Plane_CAS_01_F {
-        hit = 70; // default: 150
-        indirectHit = 11; // default: 4
+        hit = 65; // default: 150
+        indirectHit = 10; // default: 4
         indirectHitRange = 3; // default: 3
     };
+
+    // adjust damage and splash damage, AP Rounds
+    class Cannon_30mm_HE_Plane_CAS_03_F: Gatling_30mm_HE_Plane_CAS_01_F {
+        hit = 75; // default: 150
+        indirectHit = 1; // default: 4
+        indirectHitRange = 0.25; // default: 3
+        deflecting = 5;
+    };
+
+    class SubmunitionBase;
+    class ACE_30mm_mix: SubmunitionBase {
```

Comment:
Can we get a more descriptive classname please? Also, wouldn't `SubmunitionBullet` be a better base class?